### DISCUSSION TITLE
Using Plotly.Fx.hover in callbacks

### DISCUSSION THREAD

**Discussion by paradox460** [dbuezas/lovelace-plotly-graph-card#396] [Q&A]
I'm trying to make a stacked graph initiate hover events on _all_ graphs when you hover over one graph. I figure I can do this with an `on_hover` callback, and a function, but I don't see how I can use the `Plotly.Fx` module; it does not appear to be available when probing from a browser's `debug` call or just calling directly in the `$fn`

Basically I'm trying to recreate the effect seen https://plotly.com/javascript/hover-events/#coupled-hover-events, but for a stacked grid of plots

**Comment by dbuezas** [comment]
Hi there 👋 
That's unfortunately not something I've considered, so the Plotly object is not exposed to the $fns. 
I'll transfer this to a feature request issue.

**Comment by dbuezas** [comment]
https://github.com/dbuezas/lovelace-plotly-graph-card/issues/398
